### DISCUSSION TITLE
Recognize workflow images created by MS visio as text

### DISCUSSION THREAD

**Discussion by a4073631** [jsvine/pdfplumber#1070] [Q&A]
thank you for providing a good pdf parser.

If a figure is not saved as an image in a pdf, it will be extracted as text. ex) visio

Is there a way to extract pdf of these structures as an image as well?

Attached is the problematic pdf.

## Code to reproduce the problem

import pdfplumber
pdf = pdfplumber.open('hi_test.pdf')
page = pdf.pages[0] 
print(page.images)
output page.images= []

## PDF file
[hi_test.pdf](https://github.com/jsvine/pdfplumber/files/13608305/hi_test.pdf)

## Environment

- pdfplumber version: 0.10.3
- Python version: 3.9.0
- OS: window 10

**Comment by jsvine** [comment]
Hi @a4073631, sure, `page.to_image()` will turn any page into an image. If you'd like to select only part of the page, you can use `page.crop((x0, top, x1, bottom)).to_image()`. The default resolution is 72 DPI; if you'd like to use a higher resolution, you can do so, e.g.: `page.to_image(resolution=150)`.
